scss/detail.scss

```diff
@@ -0,0 +1,89 @@
+@import 'colors';
+
+%display-config-vertical {
+    display: flex;
+    align-items: center;
+}
+
+%conteiner-book-elements-height {
+    max-height: 90%;
+    height: 90%;
+}
+
+* {
+    font-family: 'Roboto', sans-serif;
+}
+
+.general-container {
+    background-color: $general-background-color;
+    @extend %display-config-vertical;
+    justify-content: center;
+    height: 100vh;
+}
+
+.container-book {
+    height: 60%;
+    width: 75%;
+    background-color: white;
+    box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+    @extend %display-config-vertical;
+    justify-content: flex-start;
+}
+
+.book-cover {
+    @extend %conteiner-book-elements-height;
```

Comment:
unnecessary property, when you work with images is a good practice to set only with o height.